src/main/scala/org/zalando/spearheads/innkeeper/routes/RequestParameters.scala

```diff
@@ -0,0 +1,18 @@
+package org.zalando.spearheads.innkeeper.routes
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import scala.util.Try
+
+/**
+ * @author Alexey Venderov
+ */
+object RequestParameters {
+
+  val urlDateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE_TIME
+
+  def dateTimeParameter(dateTimeParameter: String): Option[LocalDateTime] =
```

Comment:
if you want to dive deeper into `akka-http` you can consider defining a `Directive` to extract dates :8ball: